python/social_bot/worlds/pioneer2dx_camera.world

```diff
@@ -2,13 +2,13 @@
 <sdf version="1.5">
   <world name="default">
     <physics type="ode">
-      <max_step_size>0.001</max_step_size>
+      <max_step_size>0.005</max_step_size>
       <real_time_factor>1</real_time_factor>
       <real_time_update_rate>0</real_time_update_rate>
     </physics>
     <!-- Ground -->
     <include>
-      <uri>model://texture_ground_plane</uri>
+      <uri>model://ground_plane</uri>
```

Comment:
why do you change to ground_plane from texture_ground_plane?

Reply:
Will change it back.
I don't know why but i have a problem loading the texture ground plane, gzclient takes
 quite a long time to display. (Waiting downloading model until timeout). I removed this for testing and forgot changing it back it before commit.